lms/templates/courseware/progress_graph.js

```diff
@@ -100,7 +101,7 @@ $(function () {
   extraColorIndex = len(categories) #Keeping track of the next color to use for categories not in categories[]
   
   if show_grade_breakdown:
-    for section in grade_summary['grade_breakdown'].itervalues():
+    for section in six.itervalues(grade_summary['grade_breakdown']):
```

Comment:
I am debugging this line. May be here it converts the data into dict_value and pass further to js_utils

Reply:
@feanil I made change [here](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/progress_graph.js#L103) and now it returns dict_values which is further using at [line](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/progress_graph.js#L140)